lib/media/vtt_text_parser.js

```diff
@@ -39,6 +39,12 @@ shaka.media.VttTextParser =
     function(data, offset, segmentStartTime,
              segmentEndTime, useRelativeCueTimestamps) {
 
+  if (segmentStartTime > 0) {
```

Comment:
If the flag is set to true, then somebody is using it explicitly to get the new behavior, so they don't really need a warning.  At least, that's my take on it.